webApps/apps.json

```diff
@@ -259,6 +279,92 @@
           "url": "https://github.com/opencloud-eu/web-app-skeleton"
         }
       ]
+    },
+    {
+      "id": "com.github.opencloud-eu.web-extensions.maps",
+      "name": "Maps",
+      "subtitle": "Open .gpx files and view location data of geotagged images in the sidebar.",
+      "license": "AGPL-3.0",
+      "versions": [
+        {
+          "version": "1.0.0",
+          "minOpenCloud": "3.5.0",
+          "url": "https://github.com/opencloud-eu/web-extensions/releases/download/maps-v1.0.0/maps-1.0.0.zip"
+        },
+        {
+          "version": "0.1.0",
+          "minOpenCloud": "1.0.0",
+          "url": "https://github.com/opencloud-eu/web-extensions/releases/download/maps-v0.1.0/maps-0.1.0.zip"
+        }
+      ],
+      "authors": [
```

Comment:
Is this correct ?

Reply:
Good question! @dschmidt do you want to be listed as the author for the `maps` app?

And what about the `arcade` app? @fschade Do you want to be listed as author for it?

Reply:
Sure; Why not :) THX

Reply:
Wouldn't complain, but I don't insist on it... :)